### PR TITLE
Fixes #28068 - foreman dynflow-sidekiq service package

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Added buster.

**Comment by mmoll** [comment]
Is the old dynflow service ready to get deleted already?

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-foreman/pull/761 should get in first.

**Comment by ezr-ondrej** [comment]
> [theforeman/puppet-foreman#761](https://github.com/theforeman/puppet-foreman/pull/761) should get in first.

But the debian tests fail there because of the missing package `foreman-dynflow-sidekiq` :thinking:

File: debian/buster/foreman/rules [hunk 1]
```diff
@@ -28,8 +31,8 @@ build:
 
 override_dh_installinit:
 	dh_installinit
-	dh_installinit --name=dynflowd
 
 override_dh_systemd_enable:
 	dh_systemd_enable
-	dh_systemd_enable --name=dynflowd
+	dh_systemd_enable --name=dynflow-sidekiq@orchestrator
```
Comment (evgeni) [inline debian/buster/foreman/rules]:
```suggestion
	dh_systemd_enable --name=dynflow-sidekiq@orchestrator --package=foreman-dynflow-sidekiq
```

File: debian/buster/foreman/rules [hunk 2]
```diff
@@ -28,8 +31,8 @@ build:
...
+	dh_systemd_enable --name=dynflow-sidekiq@worker
```
Comment (evgeni) [inline debian/buster/foreman/rules]:
```suggestion
	dh_systemd_enable --name=dynflow-sidekiq@worker --package=foreman-dynflow-sidekiq

```

File: debian/buster/foreman/rules [hunk 3]
```diff
@@ -28,8 +31,8 @@ build:
 
 override_dh_installinit:
 	dh_installinit
-	dh_installinit --name=dynflowd
 
 override_dh_systemd_enable:
 	dh_systemd_enable
-	dh_systemd_enable --name=dynflowd
```
Comment (ekohl) [inline debian/buster/foreman/rules]: While making the change, perhaps keep dynflowd in for a bit so we can build it in nightly and test the installer PR? Then have a simple drop PR like in https://github.com/theforeman/foreman-packaging/pull/4442.

Comment (ezr-ondrej) [reply debian/buster/foreman/rules]: OK :) sounds reasonable. Should I keep enabling both services, or just `dynflowd` then?

Comment (ekohl) [reply debian/buster/foreman/rules]: I think this implementation is correct for now and allows us to test it.

File: debian/bionic/foreman/foreman-dynflow-sidekiq.install [hunk 4]
```diff
@@ -0,0 +1,4 @@
+extras/systemd/dynflow-sidekiq@.service /lib/systemd/system
+bundler.d/dynflow_sidekiq.rb /usr/share/foreman/bundler.d
+config/dynflow/orchestrator.yml etc/foreman
```
Comment (ekohl) [inline debian/bionic/foreman/foreman-dynflow-sidekiq.install]: Shouldn't this be `etc/foreman/dynflow` (same below):
```suggestion
config/dynflow/orchestrator.yml etc/foreman/dynflow
```

Comment (ezr-ondrej) [reply debian/bionic/foreman/foreman-dynflow-sidekiq.install]: It definitely should :( done.

**Comment by ekohl** [comment]
This looks correct:

```
root@804e7cd158a0:/# dpkg -l foreman-dynflow-sidekiq
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                    Version                                 Architecture Description
+++-=======================-=======================================-============-======================================================
ii  foreman-dynflow-sidekiq 9999-buster+scratchbuild+20200117121950 all          package providing dynflow sidekiq services for Foreman
root@804e7cd158a0:/# dpkg -L foreman-dynflow-sidekiq
/.
/etc
/etc/foreman
/etc/foreman/dynflow
/etc/foreman/dynflow/orchestrator.yml
/etc/foreman/dynflow/worker.yml
/lib
/lib/systemd
/lib/systemd/system
/lib/systemd/system/dynflow-sidekiq@.service
/usr
/usr/share
/usr/share/doc
/usr/share/doc/foreman-dynflow-sidekiq
/usr/share/doc/foreman-dynflow-sidekiq/changelog.Debian.gz
/usr/share/doc/foreman-dynflow-sidekiq/changelog.gz
/usr/share/doc/foreman-dynflow-sidekiq/copyright
/usr/share/foreman
/usr/share/foreman/bundler.d
/usr/share/foreman/bundler.d/dynflow_sidekiq.rb
```

**Pull request merged**